renderer/components/App.jsx

```diff
@@ -58,30 +58,7 @@ export default class App extends React.Component {
     };
 
     this.fileTreeInit();
-    this.clickHandler = this.clickHandler.bind(this);
-    this.setFileTree = this.setFileTree.bind(this);
-    this.dblClickHandler = this.dblClickHandler.bind(this);
-    this.setActiveTab = this.setActiveTab.bind(this);
-    //this.isFileOpened = this.isFileOpened.bind(this);
-    this.saveTab = this.saveTab.bind(this);
-    this.closeTab = this.closeTab.bind(this);
-    this.openCreateMenu = this.openCreateMenu.bind(this);
-    this.closeOpenDialogs = this.closeOpenDialogs.bind(this);
-    this.createMenuHandler = this.createMenuHandler.bind(this);
-    this.createItem = this.createItem.bind(this);
-    this.findParentDir = this.findParentDir.bind(this);
-    this.deletePromptHandler = this.deletePromptHandler.bind(this);
-    this.renameHandler = this.renameHandler.bind(this);
-    this.constructComponentTreeObj = this.constructComponentTreeObj.bind(this);
-    this.handleEditorValueChange = this.handleEditorValueChange.bind(this);
-    this.openSim = this.openSim.bind(this);
-    this.closeSim = this.closeSim.bind(this);
-    this.openSimulatorInMain = this.openSimulatorInMain.bind(this);
-    this.close = this.close.bind(this)
-    this.toggleTerminal = this.toggleTerminal.bind(this);
-    this.updateFileDirectory = this.updateFileDirectory.bind(this);
-
-    //reset tabs, should store state in local storage before doing this though
+    autoBind(this)
   }
```

Comment:
convert all handlers to arrow functions and you wont need autoBind, also change all naming convention from dblClickHandler to handleDoubleClick

Reply:
Goof thinking there. I had originally turned them to arrow functions but just threw in autobind instead